pyproject.toml

```diff
@@ -4,25 +4,31 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "baserun"
-version = "0.5.5"
+version = "0.5.6"
```

Comment:
When we're ready we're going to want to bump this. This diff is just showing because of a bad merge